src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -60,6 +60,8 @@ public class RestoreArgs
 
         public int? LockFileVersion { get; set; }
 
+        public bool? ValidateRuntimeAssets { get; set; }
```

Comment:
currently yes, this is the main entry point for restore where properties can be overridden and if they are not set the restore runner will determine the defaults